configure.ac

```diff
@@ -2305,6 +2305,69 @@ then
 fi
 # }}}
 
+# --with-libdpdk {{{
+AC_ARG_WITH(libdpdk, [AS_HELP_STRING([--with-libdpdk@<:@=PREFIX@:>@], [Path to the DPDK build directory.])],
+[
+  if test "x$withval" != "xno" && test "x$withval" != "xyes"
+	then
```

Comment:
for the if statement I'm assuming?
if should be on the same indentation as then, correct?